Dockerfile

```diff
@@ -3,7 +3,8 @@ FROM php:5.6-apache
 RUN a2enmod rewrite
 
 # Install the PHP extensions we need, and other packages
-RUN apt-get update \
+RUN set -ex \
```

Comment:
Prefixing all the run statements with `set -ex` for easier debugging, as is used in some other repos